lib/c-api/src/wasm_c_api/engine/config/mod.rs

```diff
@@ -0,0 +1,54 @@
+#[cfg(feature = "sys")]
+pub mod sys;
+#[cfg(feature = "sys")]
+pub use sys::wasm_config_set_sys_compiler;
```

Comment:
I'd call this `wasm_config_sys_set_compiler`, so we can customize more settings in the future via `wasm_config_sys_*`

Reply:
[0f304c4](https://github.com/wasmerio/wasmer/pull/5443/commits/0f304c409bc12cabb57ae28af7a5bfc146120dd4)